docs/content/docs/deployment/elastic_scaling.md

```diff
@@ -25,22 +25,100 @@ under the License.
 
 # Elastic Scaling
 
-Apache Flink allows you to rescale your jobs. You can do this manually by stopping the job and restarting from the savepoint created during shutdown with a different parallelism.
+Apache Flink allows you to rescale your jobs when the job is either not-catching up with inputs or over-provisioned. You can do this by manually stopping the job and restarting from the savepoint created during shutdown with a different parallelism.
 
-This page describes options where Flink automatically adjusts the parallelism instead.
+This page describes options **where Flink automatically adjusts the parallelism** instead, without relying on external orchestration.
 
-## Reactive Mode
+Elastic scaling is unlocked by two new schedulers: [Adaptive Scheduler](#adaptive-scheduler) (streaming) and [Adaptive Batch Scheduler](#adaptive-batch-scheduler) (batch).
+
+## Adaptive Scheduler
+
+The Adaptive Scheduler can adjust the parallelism of a job based on available slots. It will automatically reduce the parallelism if not enough slots are available to run the job with the originally configured parallelism; be it due to not enough resources being available at the time of submission, or TaskManager outages during the job execution. If new slots become available the job will be scaled up again, up to the configured parallelism.
+
+In Reactive Mode (see below) the configured parallelism is ignored and treated as if it was set to infinity, letting the job always use as many resources as possible.
+
+One benefit of the Adaptive Scheduler over the default scheduler is that it can handle TaskManager losses gracefully, since it would just scale down in these cases.
+
+{{< img src="/fig/adaptive_scheduler.png" >}}
+
+Adaptive Scheduler builds on top of a feature called [Declarative Resource Management](https://cwiki.apache.org/confluence/display/FLINK/FLIP-138%3A+Declarative+Resource+management). As you can see, instead of asking for the exact number of slots, JobMaster declares its desired resources (for reactive mode the maximum is set to infinity) to the ResourceManager, which then tries to fulfill those resources.
+
+{{< img src="/fig/adaptive_scheduler_rescale.png" >}}
+
+When JobMaster gets more resources during the runtime, it will automatically rescale the job using the latest available savepoint, eliminating the need for an external orchestration.
+
+The obvious drawback here is, that the resources are declared once the job starts and cannot be changed during the runtime. This means that the Adaptive Scheduler cannot handle cases where the job needs to be rescaled due to a change in the input rate, or a change in the performance of the workload. Starting from **Flink 1.18.x**, this is addressed by [External Declarative Resource Management](#externalized-declarative-resource-management).
```

Comment:
We can phrase this more positively. "Since 1.18.x you are also able to change the target parallelism via [EDRM..., effectively allowing you to rescale jobs at runtime."